issue_test.go

```diff
@@ -0,0 +1,31 @@
+package codacytool
+
+import (
+	"fmt"
+	"testing"
+)
+
+func testIssue() Issue {
+	file := "src/foo.go"
+	line := 5
+	message := "Foo bar"
+	patternID := "foo"
+	return Issue{
+		File:      file,
+		Line:      line,
+		Message:   message,
+		PatternID: patternID,
+	}
+}
+func TestIssueToJSON(t *testing.T) {
+	issue := testIssue()
+
+	issueAsJSON, err := issue.ToJSON()
+	if err != nil {
+		t.Error("Failed converting to JSON")
+	}
+	expectedJSON := fmt.Sprintf(`{"patternId":"%s","filename":"%s","line":%d,"message":"%s"}`, issue.PatternID, issue.File, issue.Line, issue.Message)
+	if string(issueAsJSON) != expectedJSON {
```

Comment:
please use an assertion library or a more advanced test suite for go

Reply:
Done ✅ 
Used testify library